STATUS: INLINE SUCCESSFUL SETUPTEARDOWN
MODIFIED: org.apache.shardingsphere.elasticjob.cloud.scheduler.env.BootstrapEnvironmentTest.assertWithoutEventTraceRdbConfiguration
MODIFIED FILE: /home/ecelrc/students/cli7/tool/automated_script/apache/shardingsphere-elasticjob/elasticjob-cloud/elasticjob-cloud-scheduler/src/test/java/org/apache/shardingsphere/elasticjob/cloud/scheduler/env/BootstrapEnvironmentTest.java
POLLUTER: org.apache.shardingsphere.elasticjob.cloud.scheduler.env.BootstrapEnvironmentTest.assertGetEventTraceRdbConfigurationMap
POLLUTER FILE: /home/ecelrc/students/cli7/tool/automated_script/apache/shardingsphere-elasticjob/elasticjob-cloud/elasticjob-cloud-scheduler/src/test/java/org/apache/shardingsphere/elasticjob/cloud/scheduler/env/BootstrapEnvironmentTest.java
ORIGINAL POLLUTER SIZE: 13
NEW POLLUTER SIZE: 5
ELAPSED TIME: 879.2009999752045
==========================
@@ -82,0 +82,7 @@
+ {
+     Properties properties = new Properties();
+     properties.setProperty(BootstrapEnvironment.EnvironmentArgument.EVENT_TRACE_RDB_DRIVER.getKey(), "org.h2.Driver");
+     properties.setProperty(BootstrapEnvironment.EnvironmentArgument.EVENT_TRACE_RDB_URL.getKey(), "jdbc:h2:mem:job_event_trace");
+     properties.setProperty(BootstrapEnvironment.EnvironmentArgument.EVENT_TRACE_RDB_USERNAME.getKey(), "sa");
+     ReflectionUtils.setFieldValue(bootstrapEnvironment, "properties", properties);
+ }
